src/main/java/frc/robot/subsystems/turretSubsystem/TurretCommandFactory.java

```diff
@@ -137,7 +138,34 @@ public Command zeroTurretCommand() {
 
   public Command getAimTurretCommand() {
     if (subsystem == null) return new StartEndCommand(() -> {}, () -> {});
-    return Commands.either(shortRangeShot(), longRangeShot(), () -> false);
+
+    Supplier<Command> aimCommandSupplier =
+        () -> {
+          switch (RSM.getShooterMode()) {
+            case LOB:
+              return lobShot();
+            case LONG_RANGE:
+              return longRangeShot();
+            case SHORT_RANGE:
+              return shortRangeShot();
+            default:
+              Runnable errMsg = () -> DriverStation.reportError("Unknown Shooter Mode!", true);
+              return new InstantCommand(errMsg);
```

Comment:
What happens when you run this error command? I think it would be preferable to return a valid default, either `LONG_RANGE` or `SHORT_RANGE`. Could we have a single command which runs `reportError`, and _then_ runs the default shooter aim mode command?

Reply:
Update: after talking with Casey about this, it makes more sense to report the error immediately, then return a good default value (short range). I'll make these changes.